amundsen_application/static/js/components/Tags/TagInfo/index.tsx

```diff
@@ -59,7 +59,11 @@ export class TagInfo extends React.Component<TagInfoProps> {
 }
 
 export const mapDispatchToProps = (dispatch: any) => {
-  return bindActionCreators({ submitSearch }, dispatch);
+  return bindActionCreators({
+    /* Note: Pattern intentionally isolates component from extraneous hardcoded parameters */
+    /* Note: This will have to be extended to all resources that support tags */
+    searchTag: (tagName: string) => setSearchInputByResource({ 'tag': tagName }, ResourceType.table, 0, '')
```

Comment:
I like this pattern. Although in the future we'll need to specify the resource type for Dashboards. Guess you can add `ownProps` here?

Reply:
Yes we'll have to modify something or the other that will essentially do this for all reasources that support tags.